doc/Dhyana/dhyana.rst

```diff
@@ -1,46 +1,79 @@
-.. _lima-tango-dhyana:
+Dhyana
+-------------
 
-Dhyana Tango C++ device
-==========================
+Introduction
+============
+| This device supports several camera model
+| Currently, two kinds of camera are supported: 
+| - The Dhyana95/Dhyana95 V2
+| - The Dhyana4040
 
-This is the reference documentation of the Dhyana Tango C++ device.
++-----------------------------+-----------------------+
+|                    Supported Camera                 |
++=============================+=======================+
+| .. image:: Dhyana95.png     |                       |
+|     :height: 200            | Dhyana95              |
+|     :width: 250             |                       |
++-----------------------------+-----------------------+
+| .. image:: Dhyana95V2.png   |                       |
+|     :height: 200            | Dhyana95V2            |
+|     :width: 250             |                       |
++-----------------------------+-----------------------+
+| .. image:: Dhyana4040.png   |                       |
+|     :height: 200            | Dhyana4040            |
+|     :width: 250             |                       |
++-----------------------------+-----------------------+
 
-This camera can manage 3 trigger modes : INTERNAL_SINGLE, EXTERNAL_MULTI, EXTERNAL_GATE
+Dhyana95
+-------------
 
-This camera can manage ROI 
+Introduction
+============
+| This is the reference documentation of the Dhyana Tango C++ device.
```

Comment:
This is the documentation of the Lima/Dhyana Tango C++ device.

Reply:
Ok